libs/core/src/components/sage-switch/sage-switch.scss

```diff
@@ -0,0 +1,165 @@
+:host {
+  /**
+   * @prop --switch-label-gap: Space between switch input and label
+   * @prop --switch-height: Vertical height of switch input
+   * @prop --switch-width: Horizontal width of switch input
+   * @prop --switch-message-offset: Calculated space used to align message with label
+   * @prop --switch-toggle-offset: Offset for the toggle control from its container
+   * @prop --switch-toggle-size: Calculated size of the toggle control
```

Comment:
Are these custom properties that will allow the customer to modify? If so, carry on.

Reply:
Yes, that was my assumption for the sizing items at least. I wasn't sure how deep to go for potential theming options and added a question in the consistency meeting doc.